lib/src/pubspec.dart

```diff
@@ -391,6 +391,7 @@ class Pubspec {
     if (!allowPreReleaseSdk) return false;
     if (!sdk.version.isPreRelease) return false;
     if (sdkConstraint.includeMax) return false;
+    if (sdkConstraint.min?.isPreRelease ?? false) return false;
```

Comment:
Maybe it's just me, but I think this would be easier to read as the more explicit:

```dart
if (sdkConstraint.min != null && sdkConstraint.min.isPreRelease) {
  return false;
}
```

Reply:
Done.